zebrad/src/components/sync/downloads.rs

```diff
@@ -259,15 +286,23 @@ where
         network: ZN,
         verifier: ZV,
         latest_chain_tip: ZSTip,
+        past_lookahead_limit_sender: watch::Sender<bool>,
```

Comment:
optional nit: an `AtomicBool` would be simpler (and insignificantly faster).

Reply:
I tried this, but it needed an `Arc<AtomicBool>`, and making decisions around compare/exchange and atomic orderings.

It seemed simpler to do it this way.